### ISSUE TITLE
defaultCountry does not work

### ISSUE THREAD

**Issue by ntenisOT** [mrmarkfrench/country-select-js#5]
Please check if defaultCountry works. I think it doesn't work.

defaultCountry
Type: String Default: ""
Set the default country by it's country code. You can also set it to "auto", which will lookup the user's country based on their IP address. Otherwise it will just be the first country in the list.

I use the following and it does not work: 

    $("[id$=country]").countrySelect({
        defaultCountry: function(callback) {
            $.get('http://ipinfo.io', function() {}, "jsonp").always(function(resp) {
                var countryCode = (resp && resp.country) ? resp.country : "";
                console.log(countryCode);
                countryCode.toLowerCase();
                callback(countryCode);
            });
        }
    });

and the following:

    $("[id$=country]").countrySelect({
        defaultCountry: 'auto',
        geoIpLookup: function(callback) {
            $.get('http://ipinfo.io', function() {}, "jsonp").always(function(resp) {
                var countryCode = (resp && resp.country) ? resp.country : "";
                console.log(countryCode);
                countryCode.toLowerCase();
                callback(countryCode);
            });
        }
    });

**Comment by mrmarkfrench** [comment]
Could you please provide some additional information here?

 * What value did you pass?
 * What did you expect to happen?
 * What actually happened?

**Comment by ntenisOT** [comment]
Hello thanks for the really fast answer. 

I used auto first and then I thought if you got that from the existing plugin with phone numbers the code I provided should work. But it doesn't.

**Comment by mrmarkfrench** [comment]
When you say "it didn't work", what actually happened? Did it select a country, but not the right one? Did it not select any country?

**Comment by ntenisOT** [comment]
I got an exception:

    Uncaught TypeError: Cannot read property 'iso2' of null [ line 182 ]

**Comment by mrmarkfrench** [comment]
Thanks. I'll look into that.

**Comment by ntenisOT** [comment]
Does 'auto' work for you?

**Comment by mrmarkfrench** [comment]
I've just fixed the underlying issue (that if you select a default country that it isn't able to find, it fails).

I've also removed the claim from the documentation that "auto" feature will work. It's now supported by the International Telephone plugin that this one is based on, but it's not actually implemented here. Even if it was, the documentation here doesn't provide nearly enough information about supplying the custom GeoIp lookup function that would be required.

Version 1.0.5 now fixes the error (and the pushed tag should have updated Bower and NPM to match).
